### PR TITLE
refactor(iteration-dropdown): Match Visuals of iteration dropdown in detail page

### PR DESCRIPTION
**Pull request by SMahil**
Match Visuals of iteration dropdown in detail page

Visuals : https://redhat.invisionapp.com/share/W59QILV9T#/screens/211643036

Closes: #1273

### CODE REVIEW

File: src/app/work-item/work-item-detail/work-item-detail.component.html [hunk 1]
```diff
@@ -233,30 +233,37 @@ <h2 *ngIf="loggedIn && workItem.id" id="wi-detail-title-click" class="detail-tit
...
+            <input type="hidden" name="" value="">
+            <span *ngIf="!searchIteration" (click)="lookupIterations()" class="pointer"
+            [class.none-assigned]="!workItem.relationships?.iteration?.data?.attributes?.name"
+             id="iteration-assocaited">
               {{ workItem.relationships?.iteration?.data?.attributes?.name || "None" }}
             </span>
-            <span *ngIf="searchIteration"
-              id="WI_details_iteration" dropdown>
-              <button class="btn btn-default" type="button" #dropdownButton
-                id="WI_details_iteration_dropdownbtn" data-toggle="dropdown" dropdownToggle>
```
Comment (naina-verma) [inline src/app/work-item/work-item-detail/work-item-detail.component.html]: We need this ID to support our test cases, can you please add this ID in your new code!

**Comment by naina-verma** [comment]
Video :https://drive.google.com/a/redhat.com/file/d/0B1qUq_laG6COdV9nQkM0ZjE5T0k/view?usp=sharing
this is going wrong : Overlap or iteration and Area

**Comment by SMahil** [comment]
@naina-verma Overlapping area over iteration will be resolved while updating area.

**Comment by michaelkleinhenz** [comment]
Can this be merged? @SMahil

**Comment by SMahil** [comment]
@michaelkleinhenz yes we can

**Comment by nimishamukherjee** [comment]
The overlap of Area and Iteration will be taken up in a separate PR.

**Comment by nimishamukherjee** [comment]
[test]

**Comment by naina-verma** [comment]
@nimishamukherjee , @SMahil 
Regarding this "The overlap of Area and Iteration will be taken up in a separate PR."
In that case , This PR is effecting Functional tests cases, before merging this PR all Functional tests should be in working condition.
Otherwise this PR might create more regression. 
Can some one do Fix PR on @SMahil's branch !!!

**Comment by SMahil** [comment]
@naina-verma this has dependency on issue no #1443

Will submit PR on that first.

**Comment by SMahil** [comment]
This PR is covered in #1530 as both have tests dependency on each other

**Pull request closed without merging**